### PR TITLE
docs(efb): GSX Integration

### PR DESCRIPTION
**Pull request by Lucky38i**
## Summary

Documentation for: https://github.com/flybywiresim/a32nx/pull/7695
### Location

fbw-a32nx/feature-guides/flypados3/settings/
fbw-a32nx/feature-guides/flypados3/ground/

Discord username (if different from GitHub): Lucky38i

### CODE REVIEW

File: docs/fbw-a32nx/feature-guides/flypados3/ground.md [hunk 1]
```diff
@@ -188,6 +206,29 @@ The current cabin and cargo area layout of the A32NX consists of:
     - Aft Baggage
     - Aft Bulk Loose
 
+### GSX Integration
+
+!!! warning ""
+    You must enable [Payload Sync](settings.md#3rd-party-options) in order for this to function.
+
+#### Caveats
+- You won't witness passengers boarding the aircraft until, physical passengers from GSX enter the aircraft. You'll notice this when you request boarding through the GSX menu but nothing appears to happens in the EFB just yet.
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/ground.md]:
```suggestion
- You won't witness passengers boarding the aircraft until physical passengers from GSX enter the aircraft. You'll notice this when you request boarding through the GSX menu but nothing appears to happen in the EFB just yet.
```

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 2]
```diff
@@ -216,6 +216,35 @@ Settings for realism aspects of the A32NX aircraft.
         ![img_3.png](../../assets/flypados3/flypad-settings-realism-avatars.png){loading=lazy align=center width=49%}
         ![img_2.png](../../assets/flypados3/flypad-settings-realism-avatars2.png){loading=lazy align=center width=49%}
   
+
+## 3rd Party Options
+
+Settings for integrations with various 3rd party applications
+// TODO Add an image here
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md]:
```suggestion
```

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md [hunk 3]
```diff
@@ -216,6 +216,35 @@ Settings for realism aspects of the A32NX aircraft.
...
+    <a href="../online-atc/">  <div class="imagemap" style="position: absolute; left: 1.7%; top: 38.0%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Online ATC</span></div></a>
+    <a href="../failures/">    <div class="imagemap" style="position: absolute; left: 1.7%; top: 45.1%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Failures</span></div></a>
+    <a href="../checklists/">  <div class="imagemap" style="position: absolute; left: 1.7%; top: 51.3%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Checklists</span></div></a>
+    <a href="../presets/">     <div class="imagemap" style="position: absolute; left: 1.7%; top: 58.7%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Presets</span></div></a>
+    <a href="../settings/">    <div class="imagemap" style="position: absolute; left: 1.7%; top: 85.0%; width: 5.8%; height: 7.0%;"><span class="imagemapname">Settings</span></div></a>
+    <span class="imagesub">Click on the menu icons in this image to see other flyPad pages.</span>
+</div>
```
Comment (Valastiri) [inline docs/fbw-a32nx/feature-guides/flypados3/settings.md]: I guess you wouldn't need this considering how far down the page it. Can also leave it no harm.

**Review by Valastiri — CHANGES_REQUESTED** [review]
Minor comments.

Thanks for adding /venv/ into the gitignore. I thought I had but realized I named my venv's differently heh. I am under the assumption everything is already tested with GSX?

File: docs/fbw-a32nx/feature-guides/flypados3/settings.md (hunk 3 above)
Comment (Lucky38i) [reply docs/fbw-a32nx/feature-guides/flypados3/settings.md]: I just did it cause all the other entries did

**Review by Valastiri — APPROVED** [review]

**Review by frankkopp — CHANGES_REQUESTED** [review]
The image is missing the window frame - check out the other images for comparison.
You need to also exchange the main image of the settings page

**Review by frankkopp — APPROVED** [review]

**Pull request merged**